packages/speech/package.json

```diff
@@ -51,36 +50,28 @@
     "google cloud",
     "cloud",
     "google speech",
-    "speech"
+    "speech",
+    "Google Cloud Speech API"
   ],
   "dependencies": {
-    "@google-cloud/common": "^0.13.0",
-    "@google-cloud/common-grpc": "^0.3.0",
-    "events-intercept": "^2.0.0",
-    "extend": "^3.0.0",
-    "google-gax": "^0.13.0",
     "google-proto-files": "^0.12.0",
-    "is": "^3.1.0",
-    "propprop": "^0.3.1",
+    "google-gax": "^0.13.2",
+    "extend": "^3.0.0",
     "pumpify": "^1.3.5",
-    "request": "^2.74.0",
     "stream-events": "^1.0.1",
-    "string-format-obj": "^1.1.0",
-    "through2": "^2.0.1"
+    "through2": "^2.0.3"
   },
   "devDependencies": {
-    "@google-cloud/storage": "*",
-    "async": "^2.0.1",
-    "methmeth": "^1.1.0",
-    "mocha": "^3.0.2",
-    "proxyquire": "^1.7.10",
-    "tmp": "^0.0.31",
-    "uuid": "^3.0.1"
+    "intelli-espower-loader": "^1.0.1",
+    "mocha": "^3.2.0",
+    "nyc": "^10.3.0",
```

Comment:
I don't think we need this, since we're not using it from a `cover` task specifically inside this module.